courses/beginner/03.md

```diff
@@ -120,14 +120,24 @@ Because InSpec resembles human-readable language, you might read this test as "/
 
 #### `file`
 
-[file](https://www.inspec.io/docs/reference/resources/file/) is an InSpec [resource](https://www.inspec.io/docs/reference/resources/). If you're familiar with Chef, you know that a resource configures one part of the system. InSpec resources are similar. For example, the InSpec file resource tests for file attributes, including a file's owner, mode, and permissions. The example examines the /tmp directory.
+[file](https://www.inspec.io/docs/reference/resources/file/) is an InSpec [resource](https://www.inspec.io/docs/reference/resources/). Resources are written as Ruby classes to describe a part of the system, providing attributes that are easy to call upon within the InSpec test. For example, the InSpec file resource tests for file attributes, including a file's owner, mode, and permissions. The example examines the /tmp directory.
+
+:::info Note
+If you're familiar with Chef, you know that a resource configures one part of the system. InSpec resources are similar.
+:::
 
 ---
 
 #### `it`
 
 The `it` statement validates one of your resource's features. A `describe` block contains one or more `it` statements. `it` enables you to test the resource itself. You'll also see `its`, which describes some feature of the resource, such as its mode or owner. You'll see examples of both `it` and `its` shortly.
 
+:::warning it vs. its
+Important! Just like in English grammar, pay attention to the difference between the thing (it) and the possessive word (its). `it` would be followed by something like `should`. `its` would be followed by some attribute of the thing, like `signal`. Here are two examples:
+`it { should be_owned_by 'root' }` where `it` is a `file`.
+`its("signal") { should eq "on" }` where `signal` is an attribute that belongs to a specific config resource.
```

Comment:
Maybe the following sentences can be modified:

> `it` would be followed by something like `should`. `its` would be followed by some attribute of the thing, like `signal`.

to something like:

> `it` would be followed by an expected state/value of the resource. `its` would be followed by an expected state/value of **the attribute** of the resource.

Reason for change:

1. Keeping it generic before the examples could highlight the concept better
2. When reading the sentences, I thought it could be implied that its statements wouldn't have should - until I read the examples

Reply:
Agreed. Updating.